renderer/Image.h

```diff
@@ -256,6 +256,16 @@ class idImage {
 	imageBlock_t		cpuData;				// CPU-side usable image data (usually absent)
 	imageResidency_t	residency;				// determines whether cpuData and/or texnum should be valid
 	imageLoadState_t	backgroundLoadState;	// state of background loading (usually disabled)
+
+	// START bindless texture support
+private:
+	GLuint64			textureHandle;
+	bool				isGpuResident;
```

Comment:
Unfortunately, that is the nomenclature with bindless textures, you have to make them "resident" before using them in a shader. I think it's potentially error-prone to combine it with the existing enum, but I'll rename the variable `isBindlessHandleResident`, which should hopefully make it clearer that it's a different kind of residency.

Reply:
Actually, it is not too different in theory if you think about it:

 * CPU-resident: cpu-side buffer is valid
 * GPU-resident: texture "name" is valid  (is it the right GL term? =) )
 * the-new-resident: texture bindless handle is valid

But in practice it may be not so clear. Anyway, I don't argue with leaving it as it is now, it is pretty clear what it means.